### ISSUE TITLE
[Help needed] Long tap returns wrong value in onSlidingComplete

### ISSUE THREAD

**Issue by MaximeCrp** [Sharcoux/slider#35]
Hi,
I'm using this slider as an interactive audio track progress bar, which works great in general.
I'm having a specific issue, I do not know what I'm doing wrong with the use of this component.
When the user briefly taps the slider, onSlidingComplete will return the correct selected value. 
Yet, when the user taps for a longer time (could be just a regular, not so quick tap), the returned value is the current value of the slider and not the one selected.
It also shows that way in the UI since the thumb would also stay at the previous value and not the currently selected one. Everything works correctly again if user actually slides on the slider, but not when no slide is done, only tap

**Comment by Sharcoux** [comment]
Do you have this on Web, Android, iOS? All of them?

**Comment by Sharcoux** [comment]
I could not reproduce on web...

**Comment by Sharcoux** [comment]
Seems to work fine [here](https://snack.expo.dev/@sharcoux/on-sliding-comlete) on all devices. The problem comes probably from your side.

**Comment by MaximeCrp** [comment]
Thank you for investigating. Thanks to you I realized what I was doing "wrong". 
I was not changing the slider value with `onSlidingStart`, and what this component is doing is actually displaying the `value` prop over the value of the user press (which could be perfectly good in all situations, just not what I was expecting for my use). 
I fixed this by changing the value I feed to the slider depending on whether the user is still selected the next value.
Thank you again
